geps/gep-3539/index.md

```diff
@@ -0,0 +1,200 @@
+# GEP-3539: ClusterIP Gateway - Gateway API to Expose Pods on Cluster-Internal IP Address
+
+* Issue: [#3539](https://github.com/kubernetes-sigs/gateway-api/issues/3539)
+* Status: Memorandum
```

Comment:
This should currently be `Provisional`, as it's the first iteration and we are still deciding on the approach here.

The `Memorandum` status is for registering general agreement about things, not for features that will require actual code changes to the Gateway API specification (which this definitely will).

This also needs to be changed in the corresponding `metadata.yaml` file - the YAML file is actually the canonical place for the status, this is just to remind everyone. I'll suggest the same change there.